### PR TITLE
Amazon cloud drive consistency delay

### PR DESCRIPTION
**Pull request by AndersGron**
I have no idea if this is correct or not, but it seems to fix my issues with consistency.

### CODE REVIEW

File: Duplicati/Library/Main/Operation/BackupHandler.cs [hunk 1]
```diff
@@ -1183,8 +1183,6 @@ private bool AddBlockToOutput(BackendManager backend, string key, byte[] data, i
                         
                         m_blockvolume.Dispose();
                         m_blockvolume = null;
-                        m_indexvolume.Dispose();
```
Comment (kenkendk) [inline Duplicati/Library/Main/Operation/BackupHandler.cs]: Why are these lines removed?

**Comment by kenkendk** [comment]
It looks good, it adds delays to all the operations, and I think it is consistent.

File: Duplicati/Library/Main/Operation/BackupHandler.cs (hunk 1 above)
Comment (AndersGron) [inline Duplicati/Library/Main/Operation/BackupHandler.cs]: Calling Dispose() at that point is always an error because the if block above guarantees that m_indexvolume is null at that point.
